web/themes/custom/cloudy/templates/content/construction_project/node--construction-project.html.twig

```diff
@@ -7,20 +7,32 @@
         {{ content.field_summary.0 }}
```

Comment:
Also noticed that "summary" is not required, so we had some content with the empty lead paragraph displaying. I'm going to add a temporary conditional statement and a TODO to clean this up by updating the summary field template to have classes that we target rather than explicit class overrides.